Sources/Sentry/SentryFileManager.m

```diff
@@ -787,7 +811,39 @@ - (void)createPathsWithOptions:(SentryOptions *)options
 
     NSError *error;
     SENTRY_CASSERT([fm removeItemAtPath:path error:&error],
-        @"Failed to remove launch profile marker file: %@", error);
+        @"Failed to remove launch profile config file: %@", error);
+}
+
+void
+removeAppLaunchProfilingConfigBackupFile(void)
+{
+    NSFileManager *fm = [NSFileManager defaultManager];
+    NSString *path = launchProfileConfigBackupFileURL().path;
+    if (![fm fileExistsAtPath:path]) {
+        return;
+    }
+
+    NSError *error;
+    SENTRY_CASSERT([fm removeItemAtPath:path error:&error],
+        @"Failed to remove launch profile config file: %@", error);
+}
+
+void
+saveAppLaunchProfilingConfigFile(void)
```

Comment:
`m`: The naming is a bit confusing to me. This method doesn't save anything, but instead, it moves the current config to the backup location. The name should reflect that. Something like `moveAppLaunchProfilingConfigFileToBackUp`.

Reply:
renamed to `backupAppLaunchProfilingConfigFile`